doc/src/build/install.md

```diff
@@ -141,6 +141,8 @@ Use the following command to install `libssl-dev`:
 sudo apt-get install libssl-dev
 ```
 
+If your Linux version does not have `libssl-dev` [find a correct package name here](https://index.ros.org/d/libssl-dev/).
```

Comment:
Is it that the package is not available or not supported on the Linux version? 
If the version of Linux you use doesn't support `libssl-dev`, find an equivalent package for it on the [ROS Index](https://index.ros.org/d/libssl-dev/).